README.md

```diff
@@ -45,16 +44,14 @@ architecture listed below.
 
 * CentOS 7
 * Debian 7
-* (Debian 8)
-* (Debian 9)
```

Comment:
They are only temporarily not supported and should be listed here.

Reply:
I just felt that OS support should really be binary; an OS version either is or isn't supported in the current public release. Bugs and/or planned support should be listed elsewhere.

Reply:
It is not planned support, it is temporarily not supported. Those are the OSs that we support, and by support principles we should support Debian 8 and Debian 9, but we could not. It is cleaner to list it here.

Reply:
The dropped support for Debian 8 and 9 is permanent for this particular release, and the documentation is part of the release. In this light I think we should list only the OS versions we actually do support for this particular release. If we feel squirmy about removing important OS versions from the list, we could formulate an excuse below the list.

We do have a process for updating the this list for the next release, so we should not worry about forgetting to add them back. The issues in question are tagged with "High priority", so we should not worry about forgetting to fix them for the next release either.

Reply:
This PR is for the develop branch, which will be merged to the master branch at next release. It is too early to say what OSs that we will successfully suppport then. The best guess is that we will successfully support Debian 8 and Debian 9. There is no reason to do any updates now. That should be done closer to release.

From [Support criteria](https://github.com/dotse/zonemaster/blob/master/docs/internal-documentation/maintenance/SupportCriteria.md) you can conclude that we should support Debian 8 and Debian 9, and then I think we should list them in the README. It could be listed differently than in parenthesis, but not just removed.

Reply:
Fair enough as far as I'm concerned with regard to the situation at hand.

For future releases, do you agree with the reasoning in my previous comment? In that case we could update the [release process](https://github.com/dotse/zonemaster/blob/master/docs/internal-documentation/maintenance/ReleaseProcess.md#3-update-prerequisites) document with a guideline for temporarily dropped support.

Reply:
@mattias-p, I think it was correct to list Debian 8 and Debian 9 with a note that they could not be supported (even though they were on the list of OSs that we had planned to support). That could be done in different ways, but not just removing the "Debian 8" and "Debian 9" as if they were not meant to be supported.